Find a functionally equivalent code:android.app.Fragment.initState:COMMENT
Method Modifier: 
Comment:/**
 * Called by the fragment manager once this fragment has been removed,
 * so that we don't have any left-over state if the application decides
 * to re-use the instance.  This only clears state that the framework
 * internally manages, not things the application sets.
 */

@@ -9,14 +9,15 @@
     mRestored = false;
     mBackStackNesting = 0;
     mFragmentManager = null;
+    mChildFragmentManager = null;
     mActivity = null;
     mFragmentId = 0;
     mContainerId = 0;

